source/process/sg_grammar-spelling-mechanics.rst

```diff
@@ -0,0 +1,137 @@
+================================
+Grammar, spelling, and mechanics
+================================
+
+To maintain consistency across all Mattermost technical documentation, adhere to the guidelines here.
+
+Language and spelling
+=====================
+
+Write documents in English. Use American spelling.
+
+Paragraphs and sentences
+========================
+
+Paragraphs should express one idea or topic. Long paragraphs are sometimes difficult to read on screen, so try to keep them to 5 sentences or less. Short paragraphs are easier for people to scan quickly.
+
+Try to keep sentences to 25 words or less in length. Short, single-clause sentences are often easier to understand and easier to translate.
+
+Commas
+======
+
+As a general rule, the serial comma results in greater clarity. However, there are always edge cases where a serial comma adds confusion to a sentence. Therefore, the Mattermost documentation will use the following rule for commas:
+
+Use the serial comma unless doing so decreases clarity and understanding of the sentence.
+
+Preferred
+  The cows ran from wolves, coyotes, and mosquitoes.
+
+Avoid
+  The cows ran from wolves, coyotes and mosquitoes.
+
+Tone
+====
+
+Use a direct, impartial tone. Most readers of the documentation are looking for answers and solutions to their problems; they are not looking for entertainment.
+
+Preferred
+  If your password is rejected, check to make sure that Caps Lock is off, and then carefully type it in again. 
+
+Avoid
+  Failed sign in? No problem! Simply enter the correct password and we'll let you in right away.
+
+.. _capital:
+
+Capitalization
+==============
+
+Use title case for page titles and sentence case for section titles.
+
+Title case
+  Grammar, Spelling, and Mechanics
+
+Sentence case
+  Language and spelling
+
+Voice
+=====
+
+Use active voice in preference to passive voice. Active voice has the subject of a sentence doing the action. In passive voice, the subject has an action done to it. Use passive voice only when you want to emphasize the action more than the subject.
+
+Preferred
+  The system opens the *Status* pane.
+
+Avoid
+  The *Status* pane is opened by the system.
+
+Person
+======
+
+Use the second person and avoid the first person.
+
+Preferred
+  You can view the status in the *Status* pane.
+
+Avoid
+  We'll view the status in the *Status* pane.
+
+Numbers
+=======
+
+Use decimal numbers except when the number is the first word of a sentence. Use commas to make long numbers easier to read.
```

Comment:
Hmm... I wasn't clear the first time I read this what a "decimal number" was, and what was the alternative ("written number"?). 

I had the read the examples multiple times and go back to the original description before I figured it out. 

Maybe tweak a bit? 

I think I learned the term "decimal numbers" when I was in grade school, but haven't thought of them since then...